src/util/settings.cpp

```diff
@@ -32,6 +34,10 @@ static void MergeSettings(const Settings& settings, const std::string& section,
     if (auto* values = FindKey(settings.command_line_options, name)) {
         fn(SettingsSpan(*values), Source::COMMAND_LINE);
     }
+    // Merge in the read-write settings
```

Comment:
nit: the comment above ("Forced config > command line > ...") should be updated.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/15935#discussion_r453291296

> nit: the comment above ("Forced config > command line > ...") should be updated.

Thanks, updated